modules/specialized-guides/pages/public-cloud-guide/payg/payg-configure-the-image.adoc

```diff
@@ -0,0 +1,55 @@
+= Initial Preparation and Configuration of the AWS Image
+
+This section covers initial preparation and configuration of the image on {aws}.
+
+.The IAM role
+[IMPORTANT]
+====
+To ensure seamless operation, proper AWS permissions for the IAM role are essential. This role must be attached after the instance is launched; attaching it beforehand will result in a disconnection of {productname} after one hour. You can pre-create the role and then select it from the existing btn:[IAM role] dropdown during the instance configuration process.
```

Comment:
```suggestion
To ensure seamless operation, proper AWS permissions for the IAM role are essential. 
This role must be attached after the instance is launched; attaching it beforehand will result in a disconnection of {productname} after one hour. 
You can create the role in advance and then select it from the existing btn:[IAM role] dropdown during the instance configuration process.
```

Reply:
> This role must be attached after the instance is launched; attaching it beforehand will result in a disconnection of {productname} after one hour.

This is not 100% correct. 

Let's say, this role must be attached, otherwise you will see errors after 1 hour.
You can attach the role to a running instance or also during the start when the role is already created in the customers AWS account.
When they create the role during the creation of an instance, the role will not be attached automatically. This needs to be done afterwards.